NGitLab.Mock/Repository.cs

```diff
@@ -440,6 +440,21 @@ public Models.FileData GetFile(string filePath, string @ref)
             };
         }
 
+        internal IEnumerable<Models.Tree> GetTree()
+        {
+            var repo = GetGitRepository();
+            Commands.Checkout(repo, Project.DefaultBranch);
+
+            foreach(var fileSystemEntry in Directory.EnumerateFileSystemEntries(FullPath))
+            {
+                yield return new Models.Tree
+                {
+                    Name = Path.GetFileName(fileSystemEntry),
+                    Path = Path.GetFileName(fileSystemEntry),
```

Comment:
Type is likely:
- folder => tree
- file => blob

That should be easy to determine and set there :)